crypto/store/store_local.h

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
agreed for new files it's kinda stupid.  On the other hand:
```
; find . -name '*_l*.h' | grep locl | wc
     28      28     702
; find . -name '*_l*.h' | grep lcl | wc
     16      16     391
; find . -name '*_l*.h'  | wc
     48      48    1219
```
there is A LOT to be said for convention.